From 0d67dd33b3900638d3648d8aca15face8f529c18 Mon Sep 17 00:00:00 2001
From: Tomas Glozar <tglozar@gmail.com>
Date: Thu, 9 Nov 2023 17:32:42 +0100
Subject: [PATCH 12/73] ia64: Add futex_requeue() syscall

futex_requeue() was added after ia64 was removed from the kernel.

Fixes: 0f4b5f9722167 ("futex: Add sys_futex_requeue()")
---
 arch/ia64/kernel/syscalls/syscall.tbl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/ia64/kernel/syscalls/syscall.tbl b/arch/ia64/kernel/syscalls/syscall.tbl
index 915146e745fc..6db5e2f51435 100644
--- a/arch/ia64/kernel/syscalls/syscall.tbl
+++ b/arch/ia64/kernel/syscalls/syscall.tbl
@@ -376,3 +376,4 @@
 453	common	map_shadow_stack		sys_map_shadow_stack
 454	common	futex_wake			sys_futex_wake
 455	common	futex_wait			sys_futex_wait
+456	common	futex_requeue			sys_futex_requeue
-- 
2.25.1

